blueprints/ember-cli-addon-docs/index.js

```diff
@@ -63,9 +63,39 @@ module.exports = {
     }
 
     if (!hasPlugins) {
+      return this._chooseAddonsToInstall()
+        .then((addon) => {
+          return this.addAddonsToProject({
+            packages: [addon],
+            blueprintOptions: {
+              save: options.save
```

Comment:
What is this option doing? Was something behaving wrong when we didn't have it before?